lsinitrd.sh

```diff
@@ -96,7 +96,16 @@ while (($# > 0)); do
     shift
 done
 
-[[ $KERNEL_VERSION ]] || KERNEL_VERSION="$(uname -r)"
+if ! [[ $KERNEL_VERSION ]]; then
+    if type -P systemd-detect-virt > /dev/null 2>&1 && systemd-detect-virt -c &> /dev/null; then
+        # shellcheck disable=SC2012
+        KERNEL_VERSION="$(cd /lib/modules && ls -1 | tail -1)"
+        # shellcheck disable=SC2012
+        KERNEL_VERSION || KERNEL_VERSION="$(cd /usr/lib/modules && ls -1 | tail -1)"
+    else
+        KERNEL_VERSION="$(uname -r)"
```

Comment:
I think we want this fallback unconditionally - I can imagine cases today that are intentionally running dracut in a container but targeting the host kernel.